assets/src/edit-story/components/form/advancedDropDown/dropDown.js

```diff
@@ -185,39 +222,10 @@ function DropDown({
         <DropDownTitle>{selectedOption?.name || placeholder}</DropDownTitle>
         <DropDownIcon />
       </DropDownSelect>
-      {!disabled && (
+      {isOpen && !disabled && displayInContent && list}
+      {!disabled && !displayInContent && (
         <Popup anchor={ref} isOpen={isOpen} fillWidth={DEFAULT_WIDTH}>
-          <OptionsContainer
-            isOpen={isOpen}
-            onClose={debouncedCloseDropDown}
-            getOptionsByQuery={getOptionsByQuery}
-            hasSearch={hasSearch}
-            renderContents={({
-              searchKeyword,
-              setIsExpanded,
-              trigger,
-              queriedOptions,
-              listId,
-            }) => (
-              <List
-                listId={listId}
-                value={selectedId}
-                keyword={searchKeyword}
-                onSelect={handleSelect}
-                onClose={debouncedCloseDropDown}
-                onExpandedChange={setIsExpanded}
-                focusTrigger={trigger}
-                onObserve={onObserve}
-                options={options || queriedOptions}
-                primaryOptions={primaryOptions}
-                primaryLabel={primaryLabel}
-                priorityOptions={priorityOptions}
-                priorityLabel={priorityLabel}
-                searchResultsLabel={searchResultsLabel}
-                renderer={renderer}
-              />
-            )}
-          />
+          {list}
```

Comment:
Ah, I see the difference now. I think `isInline` would be a better name. But the previous logic should have a comment about that if the dropdown is inlined, it's not residing in a separate popup layer, and thus it's absolute rather than relative.